numpy/linalg/linalg.py

```diff
@@ -2288,15 +2288,15 @@ def lstsq(a, b, rcond="warn"):
 def _multi_svd_norm(x, row_axis, col_axis, op):
     """Compute a function of the singular values of the 2-D matrices in `x`.
 
-    This is a private utility function used by numpy.linalg.norm().
+    This is a private utility function used by ``numpy.linalg.norm()``.
```

Comment:
The double backticks mean this will be rendered as code. While correct, it would be nicer to turn it into a link to the documentation of `norm`, which can be done by using single back-ticks. Then the question is whether to use `numpy.linalg.norm`, which is the fully qualified function name, or just `linalg.norm`, since we are already in the `numpy` namespace and should work, or just `norm`, since we are already in the `linalg` namespace. I find it confusing and time-consuming to figure out which works, so going with the safe, fully-qualified `numpy.linalg.norm` is probably best.